scalatest/src/main/scala/org/scalatest/tools/HtmlReporter.scala

```diff
@@ -820,7 +820,7 @@ private[scalatest] class HtmlReporter(
     val (grayStackTraceElements, blackStackTraceElements) =
       throwable match {
         case Some(throwable) =>
-          val stackTraceElements = throwable.getStackTrace.toList
+          val stackTraceElements = throwable.getStackTrace.map(_.nn).toList
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.